modules/integration/fsx-lustre-on-eks/stack_fsx_eks.py

```diff
@@ -166,17 +175,18 @@ def __init__(
                             "restartPolicy": "Never",
                             "containers": [
                                 {
-                                    "name": "app",
-                                    "image": "centos",
-                                    "command": ["/bin/sh"],
+                                    "name": "set-permissions-job",
+                                    "image": "public.ecr.aws/amazonlinux/amazonlinux:2",
+                                    "command": ["/bin/sh", "-c"],
                                     "args": [
-                                        "-c",
-                                        "chmod 2775 /data && chown root:users /data",
+                                        f"chmod -R 777 {dra_export_path}",
+                                        f"chown -R root:users {dra_export_path}"
                                     ],
                                     "volumeMounts": [
                                         {
                                             "name": "persistent-storage",
-                                            "mountPath": "/data",
+                                            "mountPath": dra_export_path,
+                                            "subPath": dra_export_path[1:],
```

Comment:
is this for the leading `/`? probably need to check whether path contains it

Reply:
any subPath cannot start with `/`. If it starts with `/`, the pod fails

Reply:
Yes I mean if the user passes `dra_export_path=some/path` without leading `/` we will trim the first character. Worth to add a note in README.